.github/contributing.md

```diff
@@ -1,7 +1,19 @@
 # Contributing to Skript
-Skript is an open source project. That basically means that you are
-encouraged to contribute by reporting issues and writing code. Basic
-guidelines are posted below to save time from everyone.
+Skript is an open source project, and you're encouraged to contribute to it.
+Both reporting issues and writing code help us. However, please review the
+following guidelines *before* doing either of these. Properly created issues
+and pull requests are often resolved faster than those that ignore them.
+
+## Behavior
+Please treat others with respect in the issue tracker and comments of pull
+requests. We hope that you do are a decent person and do this without
+telling. Failing that, your rudeness (and associated issues) may be ignored,
```

Comment:
"We hope that you do are a decent person" 
Im not entirely sure what this line was supposed to be, but it didn't seem right to me.

Reply:
Hmm, true. It reads as condescending, more so than I intended. Anyway, the idea was to tell people who're using common sense that they have nothing to worry before threatening with bans.

Reply:
I was more referring to the extra "do" in there, I would think it should be "We hope that you are a decent person..."
I didn't even pick up on the slight condescending part haha.